dojo/settings/settings.dist.py

```diff
@@ -812,6 +812,7 @@ def generate_url(scheme, double_slashes, user, password, host, port, path, param
     'DSOP Scan': ['cve'],
     'Acunetix Scan': ['title', 'description'],
     'Trivy Scan': ['title', 'severity', 'cve', 'cwe'],
+    'TFSec Scan': ['severity', 'vuln_id_from_tool', 'file_path', 'line', 'description'],
```

Comment:
Should we remove the description? I have the feeling that if `vuln_id_from_tool` is used in de-duplication algorithm we can ignore description.

Reply:
Done